extensions/github-copilot/src/copilot-usage.tsx

```diff
@@ -0,0 +1,140 @@
+import { List, Icon, Color, Action, ActionPanel } from "@raycast/api";
+import { useCallback } from "react";
+import { useCopilotUsage } from "./hooks/useCopilotUsage";
+import { fetchCopilotUsage } from "./services/copilot";
+
+function Command() {
+  const fetchUsage = useCallback(() => fetchCopilotUsage(), []);
+
+  const { isLoading, usage, revalidate } = useCopilotUsage(fetchUsage);
+
+  const formatUsage = (current: number, limit: number | null): string => {
+    if (limit === null) {
+      return "Included";
```

Comment:
Do you think this would be clearer?

```suggestion
      return "Unlimited";
```